packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -451,4 +451,22 @@ class _RenderFlexibleSpaceHeaderOpacity extends RenderOpacity {
 
   @override
   bool get isRepaintBoundary => false;
+
+
```

Comment:
nit: remove one of the duplicated blank lines?

Reply:
Done